internal/infra/store/memory/store.go

```diff
@@ -53,51 +55,100 @@ func (s *Store) Close() {
 	close(s.quit)
 }
 
-func (s *Store) GetGauge(req *model.MetricRequest) (*model.Gauge, error) {
+func (s *Store) GetGauge(ctx context.Context, req *model.MetricsV2) (*model.Gauge, error) {
```

Comment:
```suggestion
func (s *Store) GetGauge(_ context.Context, req *model.MetricsV2) (*model.Gauge, error) {
```
иначе линтер будет ругаться.